<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>THDClass</class>
 <widget class="QMainWindow" name="THDClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>THD</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">

QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
   </property>
   <widget class="QGroupBox" name="gBoxOutputFeeds">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #FFOECE, stop: 1 #FFFFFF);
 }</string>
    </property>
    <property name="title">
     <string>Output Feeds</string>
    </property>
    <widget class="QGroupBox" name="gBoxLiveFeed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>691</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Live Feed</string>
     </property>
     <widget class="QLabel" name="lblLiveFeed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>431</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>Live Feed Empty</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblOutputInformationTitle">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Output Information:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>231</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblFramesProcessedTitle">
         <property name="text">
          <string>Frames Processed: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLabel" name="lblFramesProcessed">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblTimeElaspedTitle">
         <property name="text">
          <string>Time Elasped:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLabel" name="lblTimeElapsed">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblVideoSpeedTitle">
         <property name="text">
          <string>Video Speed:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLabel" name="lblVideoSpeed">
         <property name="text">
          <string>1x</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblVehicleSpeedTitle">
         <property name="text">
          <string>Vehicle Speed:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLabel" name="lblVehicleSpeed">
         <property name="text">
          <string>Not Callibrated</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblEngineCallibrationTitle">
         <property name="text">
          <string>Engine Callibration:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLabel" name="lblEngineCallibration">
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gBoxDetectionFeed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>691</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>Detection Feed</string>
     </property>
     <widget class="QLabel" name="lblDetectionFeed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>671</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string>Detection Feed Empty</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QSlider" name="hSliderProgress">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>691</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal {
     border: 1px solid #999999;
     height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
     background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
     margin: 2px 0;
 }

QSlider::handle:horizontal {
     background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
     border: 1px solid #5c5c5c;
     width: 18px;
     margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
     border-radius: 3px;
 }</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>320</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnSlowDown">
        <property name="text">
         <string>Slow Down</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPlayPause">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSpeedUp">
        <property name="text">
         <string>Speed Up</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gBoxAnalysisImages">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>561</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #FFOECE, stop: 1 #FFFFFF);
 }

QFrame, QLabel, QToolTip {
     border: 2px solid gray;
     border-radius: 4px;
     padding: 2px;
     background-image: url(images/welcome.png);
 }</string>
    </property>
    <property name="title">
     <string>Analysis Images</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>521</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblHazard1Title">
        <property name="text">
         <string>Hazard 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblHazard2Title">
        <property name="text">
         <string>Hazard 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblHazard3Title">
        <property name="text">
         <string>Hazard 3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>521</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lblHazard1">
        <property name="maximumSize">
         <size>
          <width>169</width>
          <height>169</height>
         </size>
        </property>
        <property name="text">
         <string>No Hazard</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblHazard2">
        <property name="maximumSize">
         <size>
          <width>169</width>
          <height>169</height>
         </size>
        </property>
        <property name="text">
         <string>No Hazard</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblHazard3">
        <property name="maximumSize">
         <size>
          <width>169</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>No Hazard</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>521</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lblAnalysisImage1">
        <property name="maximumSize">
         <size>
          <width>257</width>
          <height>159</height>
         </size>
        </property>
        <property name="text">
         <string>No Image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAnalysisImage2">
        <property name="text">
         <string>No Image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gBoxControlPanel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>420</y>
      <width>561</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #FFOECE, stop: 1 #FFFFFF);
 }

</string>
    </property>
    <property name="title">
     <string>Control Panel</string>
    </property>
    <widget class="QTabWidget" name="tWidgetControls">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>541</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane { /* The tab widget frame */
     border-top: 2px solid #C2C7CB;
     position: absolute;
     top: -0.5em;
 }

 QTabWidget::tab-bar {
     alignment: center;
 }

 /* Style the tab using the tab sub-control. Note that
     it reads QTabBar _not_ QTabWidget */
 QTabBar::tab {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
     border: 2px solid #C4C4C3;
     border-bottom-color: #C2C7CB; /* same as the pane color */
     border-top-left-radius: 4px;
     border-top-right-radius: 4px;
     min-width: 8ex;
     padding: 2px;
 }

 QTabBar::tab:selected, QTabBar::tab:hover {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                 stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
 }

 QTabBar::tab:selected {
     border-color: #9B9B9B;
     border-bottom-color: #C2C7CB; /* same as pane color */
 }</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabconsole">
      <attribute name="title">
       <string>Console</string>
      </attribute>
      <widget class="QListWidget" name="lWidgetConsole">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>541</width>
         <height>181</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::VLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabsettings">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <widget class="QLineEdit" name="lEditVideoLocation">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>511</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
     border: 2px solid gray;
     border-radius: 5px;
     padding: 0 8px;
     selection-background-color: darkgray;
 }</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblSelectVideoTitle">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Sample Feed Selection</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStart">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>140</y>
         <width>84</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBrowse">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>70</y>
         <width>84</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cBoxRGBCorrection">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>181</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Disable RGB Channel Correction</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblAdditionalSettingsTitle">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>75</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Additionl Settings:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnClear">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>70</y>
         <width>84</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cBoxDetectionPause">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>191</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Pause Upon Hazard Detection</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cBoxAnnotations">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>211</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable Scene Annotations</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>140</y>
         <width>84</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1309</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="thd.qrc"/>
 </resources>
 <connections/>
</ui>
